.github/workflows/codegen_ci.yml

```diff
@@ -117,3 +117,29 @@ jobs:
       - name: Integration specs
         run: rspec
         working-directory: ./white_label
+
+  codegen-rubocop:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2
+
+      - name: Build and install Hearth
```

Comment:
I think you can remove this - we shouldn't need hearth installed to run rubocop.

Reply:
Gotcha, I will remove now.